underscore.mdx

```diff
@@ -5,133 +5,145 @@ description: Using underscore expressions to define features
+DataFrame scoped down to the referenced columns. DataFrames can be aggregated after eligible columns are selected.
 
-Using the underscore syntax, we can define `total_mass` as follows:
+With our `Transaction` example, we can compute the number of large transactions a user as had as well as their total
+spending volume:
 
 ```diff-py
-- from chalk.features import features, has_one, has_many, DataFrame
-+ from chalk.features import _, features, has_one, has_many, DataFrame
+from chalk.features import _, features, DataFrame
 
 @features
-class Astronaut:
+class Transaction:
     id: int
-    name: str
-    height: float
-    mass: float
-    ship_id: int
+    user_id: "User.id"
+    total: float
+    sales_tax: float
+    subtotal: float = _.total - _.sales_tax
 
 @features
-class RocketShip:
+class User:
     id: int
-    engine_id: int
-
-    # `has_one` relationship with `RocketEngine`
-    engine: RocketEngine = has_one(lambda: RocketShip.engine_id == RocketEngine.id)
-
-    # `has_many` relationship with `Astronaut`
-    astronauts: DataFrame[Astronaut] = has_many(lambda: Astronaut.ship_id == RocketShip.id)
-
-+   # Defining total_mass using an underscore expression
-+   total_mass: float = _.engine.mass + _.astronauts[_.mass].sum()
+    # implicit has-many relationship with Transaction due to `user_id` above
+    transactions: DataFrame[Transaction]
+    large_transactions: DataFrame[Transaction] = _.transactions[_.total > 1000]
++   total_spend: float = _.transactions[_.total].sum()
++   num_large_transactions: int = _.large_transactions.count()
 ```
 
-Here we used the underscore syntax do a [projection](/docs/dataframe#projections) of the `mass` column in the
-astronauts `DataFrame`.
-
-
-### Filters
-
-Underscores can also be used to define [filters](/docs/dataframe#filters) on `DataFrame` features.
-
-For example, suppose we wanted to define a feature called `count_tall_astronauts` on `RocketShip`.
-We could define the feature as follows
-
-```diff-py
-@features
-class RocketShip:
-    id: int
-    engine_id: int
-
-    # `has_one` relationship with `RocketEngine`
-    engine: RocketEngine = has_one(lambda: RocketShip.engine_id == RocketEngine.id)
-
-    # `has_many` relationship with `Astronaut`
-    astronauts: DataFrame[Astronaut] = has_many(lambda: Astronaut.ship_id == RocketShip.id)
-
-    # Defining total_mass using an underscore expression
-    total_mass: float = _.engine.mass + _.astronauts[_.mass].sum()
-
-+   # Filtering on the `height` column
-+   count_tall_astronauts = _.astronauts[_.height > 2.0].count()
-```
+To compute `User.total_spend`, we needed to create a projection of the `User.transactions` DataFrame limited to only the
+`Transaction.total` column so that the `sum` aggregation could work. In contrast, no projection was needed for
+`num_large_transaction`'s `count` aggregation because `count` works on DataFrames with any number of columns.
+
+## All supported operations
```

Comment:
I thought about adding docstrings here, but I felt like it would be 90% "repeat the name of the thing as the definition", and that felt goofy